SwiftStyleKitSample/StyleKitSample/Constants.swift

```diff
@@ -8,6 +8,11 @@
 
 import Foundation
 
+struct FontProperties {
+    static let Name = "font"
+    static let Size = "size"
+}
```

Comment:
Actually, I'm not so sure. If I use enum, I will end up writing rawValue everywhere, which can be tedious. That's why I used structs with constant strings from the first place.